infrastructure/src/main/java/org/corfudb/infrastructure/orchestrator/Workflow.java

```diff
@@ -0,0 +1,28 @@
+package org.corfudb.infrastructure.orchestrator;
+
+import java.util.List;
+
+/**
+ *
+ * A workflow is an abstract container that specifies a series of steps that achieves a
+ * multi-step operation. For example, adding a new node to the cluster, which requires
+ * several steps to complete.
+ *
+ * Created by Maithem on 10/25/17.
+ */
+
+public abstract class Workflow {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "Workflow" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)

Reply:
Please convert into a interface

Reply:
Done.